.github/workflows/ci.yml

```diff
@@ -293,3 +293,14 @@ jobs:
           - typo3-version: "^12.4"
             php-version: "8.3"
             composer-dependencies: highest
+  documentation:
+    name: Documentation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Test if the documentation will render without warnings
+        run: |
+          mkdir -p Documentation-GENERATED-temp \
```

Comment:
Do we need this `mkdir` here? And if we need it, could/should we integrate it into the Composer command?

Reply:
It was necessary some time ago. I think, it is added now to the command. I removed it.